hummingbot_gateway_broker_compose/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.9"
+services:
+  bot:
+    image: hummingbot/hummingbot:latest
+    volumes:
+      - "./hummingbot_files/conf:/conf"
```

Comment:
we should add the volume for conf/connectors and conf/strategies

Reply:
Why is this necessary?
I see that you recentrly updated the deployments for this (https://github.com/hummingbot/deploy-examples/blob/main/hummingbot_gateway_compose/docker-compose.yml#L7).

Why not mount the parent dir (conf/) as before (childs should exist from the parent dir)? What is the issue that caused this change?